creator-node/src/fileManager.js

```diff
@@ -295,17 +297,36 @@ async function saveFileForMultihashToFS (serviceRegistry, logger, multihash, exp
     }
 
     // verify that the contents of the file match the file's cid
+    let fileData = null
+    let fileSize = null
+    try {
+      fileData = (await fs.readFile(expectedStoragePath)).toString().slice(0, 99)
+      fileSize = (await fs.stat(expectedStoragePath)).size
+    } catch (e) {
+      logger.warn('Could not get fs statistics for path', expectedStoragePath)
+    }
     try {
-      decisionTree.push({ stage: 'About to verify the file contents for the CID', vals: multihash, time: Date.now() })
+      decisionTree.push({ stage: 'About to verify the file contents for the CID', vals: multihash, time: Date.now(), fileData, fileSize })
```

Comment:
whats the value in logging this `fileData` each time? seems only useful when there is some error right?

Reply:
removed this